examples/plotting/zoom_map_example.py

```diff
@@ -0,0 +1,45 @@
+import matplotlib.pyplot as plt
+from matplotlib import patches
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.map
+from sunpy.data.sample import AIA_171_IMAGE
+
+###############################################################################
+# Define a region of interest.
+# A region of interest is a portion of an image that you want to operate on.
+# You can create many shape like circle, rectangle, polygon, ellipses etc.
+length = 500 * u.arcsec
+x0 = -50 * u.arcsec
+y0 = -30 * u.arcsec
+
+###############################################################################
+# We start with the sample data and
+# Create a SunPy Map and submap over the region of interest.
```

Comment:
```suggestion
```

Reply:
This is a comment for better understanding. Should i remove this as well?